tests/integration/targets/monit/files/daemon.py

```diff
@@ -0,0 +1,130 @@
+"""Generic linux daemon base class for python.
+http://www.jejik.com/files/examples/daemon3x.py
```

Comment:
I think this file needs a copyright header. I assume it is GPLv3+ licensed?

Reply:
I've no idea how to license this. It was written by an anonymous source: https://www.jejik.com/articles/2007/02/a_simple_unix_linux_daemon_in_python/

I can try the `python-daemon` library instead and see if it works on all the builds.

Reply:
I've got some builds working. I had to skip python2.6. I'll push now and see how the build does. I've also refactored the tests a bit and tried to make it more robust.